apps/condo/domains/billing/schema/BillingAccountMeterReading.test.js

```diff
@@ -33,17 +33,15 @@ describe('BillingAccountMeterReading', () => {
     test('organization integration manager: create BillingAccountMeterReading', async () => {
         const admin = await makeLoggedInAdminClient()
         const { organization, integration, managerUserClient } = await makeOrganizationIntegrationManager()
-        const [context] = await createTestBillingIntegrationOrganizationContext(managerUserClient, organization, integration)
```

Comment:
why do you want to change it to admin?

Reply:
Since organization manager no longer has access to billing entities: https://doma.atlassian.net/browse/DOMA-2348

Reply:
But he should be able to create context